api/config/runtime.exs

```diff
@@ -21,77 +21,94 @@ if System.get_env("PHX_SERVER") do
 end
 
 if config_env() == :prod do
-  database_url =
-    System.get_env("DATABASE_URL") ||
-      raise """
-      environment variable DATABASE_URL is missing.
-      For example: ecto://USER:PASS@HOST/DATABASE
-      """
+  if System.get_env("BOOTSTRAPPED") do
+    config :api, Api.Repo,
+      username: System.get_env("DB_USER"),
+      password: System.get_env("DB_PASSWORD"),
+      hostname: System.get_env("DB_HOST"),
+      database: System.get_env("DB_NAME"),
+      port: System.get_env("DB_PORT"),
+      stacktrace: true,
+      show_sensitive_data_on_connection_error: true,
+      pool_size: 10
 
```

Comment:
i added this if block that will be triggered when we are running locally via docker compose, switching on a bootstrapped var in the env file. it might be better to add another var there that says we're local?

if this block isn't hit, we'll do the other runtime config stuff that should describe how our endpoint and rds connection are configured. i know that @paulswartz had suggested putting this in `prod.exs`, but that file is evaluated at build time rather than runtime, meaning it wouldn't have access to these env vars?